theatres/anizay-2023/TCO30_Andrew_Thunderstruck.tem_anizay/edit_me/description.ext

```diff
@@ -0,0 +1,11 @@
+// Mission settings - title, description, author etc.
+OnLoadName = "Thunderstruck";
+OnLoadMission = "Assist the United Assistance after they've repelled an attack.";
+author = "Andrew Libby, Synixe Contractors";
+
+synixe_type = 1; // 0: Contract, 1: Sub-Contract, 2: Training, 3: Special
+synixe_start_time = 1830; // Start time of the mission.
```

Comment:
```suggestion
synixe_start_time = 18; // Start time of the mission.
```

We also start at the top of the hour

Reply:
noted and changed it in the description and the mission file in Eden. The start time for the mission file is 1700, and will now be ready for 1800.